backend/lucene/src/main/java/org/hibernate/search/backend/lucene/types/codec/impl/LuceneStringFieldCodec.java

```diff
@@ -25,15 +26,17 @@
 public final class LuceneStringFieldCodec implements LuceneTextFieldCodec<String> {
 
 	private final boolean sortable;
+	private final boolean searchable;
 
 	private final FieldType fieldType;
 
 	private String indexNullAsValue;
 
 	private final Analyzer analyzerOrNormalizer;
 
-	public LuceneStringFieldCodec(boolean sortable, FieldType fieldType, String indexNullAsValue, Analyzer analyzerOrNormalizer) {
+	public LuceneStringFieldCodec(boolean sortable, boolean searchable, FieldType fieldType, String indexNullAsValue, Analyzer analyzerOrNormalizer) {
```

Comment:
Here `searchable` is after `sortable`, while in every other codec it's the opposite.

Reply:
You're right. It's better to keep the order consistent.